cfgov/unprocessed/js/modules/FlyoutMenu.js

```diff
@@ -92,14 +114,17 @@ function FlyoutMenu( element ) {
       this.dispatchEvent( 'expandBegin', { target: this } );
       _isExpanded = true;
       _isAnimating = true;
-      // If transition is not supported, call handler directly (IE9/OperaMini).
-      if ( _transitionEndEvent ) {
-        _contentDom.addEventListener( _transitionEndEvent, _expandEndBinded );
+      if ( _expandTransitionMethod ) {
+        _expandTransitionMethod.call();
```

Comment:
Hm, yeah I think my original thinking was the different scope. Since `_expandTransitionMethod` is not a function defined in `FlyoutMenu`, it seemed deceptive to have it as `_expandTransitionMethod()` since that looks like every call to a private method within `FlyoutMenu`.